src/js/components/deployments/deployment-wizard/schedulerollout.js

```diff
@@ -137,8 +139,17 @@ export class ScheduleRollout extends React.Component {
     return (
       <form style={{ overflow: 'visible', minHeight: '300px', marginTop: '15px' }}>
         <Grid container alignItems="center" direction="column">
-          <Grid item style={{ width: 'min-content', marginBottom: 30 }}>
-            {self.state.isPickerOpen || start_time ? (
+          <Grid item style={{ width: 'min-content', marginBottom: self.state.isPickerOpen || start_time ? 0 : 30 }}>
+            <FormControl>
+              <InputLabel>Set a start time</InputLabel>
+              <Select onChange={event => this.handleStartChange(event.target.value)} value={start_time ? 'custom' : 0} style={styles.textField}>
+                <MenuItem value={0}>Start immediately</MenuItem>
+                <MenuItem value="custom">Schedule a start date &amp; time</MenuItem>
```

Comment:
I wonder if it should be "Schedule *the* start date &amp; time"?

Reply:
yes, I like it more, let's change it